crates/aptos/src/common/init.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    common::{
+        types::CliConfig,
+        utils::{prompt_yes, to_common_success_result},
+    },
+    op::key::GenerateKey,
+    CliResult, Error,
+};
+use aptos_crypto::{x25519::PrivateKey, ValidCryptoMaterialStringExt};
+use clap::Parser;
+
+/// Tool to initialize current directory for the aptos tool
+#[derive(Debug, Parser)]
+pub struct InitTool {}
+
+impl InitTool {
+    pub async fn execute(&self) -> CliResult {
```

Comment:
self is never used, these can all be class methods

Reply:
Will fix with all of the other commands in a future PR.